TrashMob/client-app/src/components/EventManagement/ShareToSocialsModal.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+import { Modal, Button, Tooltip, OverlayTrigger } from 'react-bootstrap';
+import { Clipboard, GeoAltFill, Clock } from "react-bootstrap-icons";
+import { FacebookShareButton, FacebookIcon, TwitterShareButton, TwitterIcon, LinkedinShareButton, LinkedinIcon, WhatsappShareButton, WhatsappIcon, EmailShareButton, EmailIcon } from 'react-share';
+import Card from 'react-bootstrap/Card';
+import * as ToolTips from '../../store/ToolTips';
+
+interface ModalProps {
+  createdEvent: any;
+}
+
+export const SocialsModal: React.FC<ModalProps> = (props) => {
+  const [show, setShow] = useState(true);
+  const [copiedLink, setCopied] = useState(false);
+  const eventLink = `${window.location.origin}/eventdetails/${props.createdEvent.id}`
+  const eventDate = new Date(props.createdEvent.eventDate).toLocaleDateString("en-us", { year: "numeric", month: "2-digit", day: "2-digit" })
+  const eventTime = new Date(props.createdEvent.eventDate).toLocaleTimeString("en-us", { hour12: true, hour: 'numeric', minute: '2-digit' })
+
+  const renderCopyToClipboardToolTip = (
+    <Tooltip id="tooltip">
+      {ToolTips.CopyToClipboard}
+    </Tooltip>
+  );
+
+  const handleClose = () => {
+    setShow(false);
+    setCopied(false);
+  }
+
+  const handleCopyLink = () => {
+    if (copiedLink) {
+      return
+    }
+
+    navigator.clipboard.writeText(eventLink);
+    setCopied(true);
+    setTimeout(() => {
+      setCopied(false);
+    }, 2000)
+  }
+
+  const getShareMsgContent = (provider: string) => {
+    return `Join my next ${provider === 'twitter' ? "@TrashMobEco" : "TrashMob.eco"} event on ${eventDate} at ${eventTime} in ${props.createdEvent.city}.\n` +
+      `Sign up using the link for more details. Help me clean up ${props.createdEvent.city}!`
+  }
+
+  const EventLink = () => {
+    return (
+      <Card className="pr-2">
+        <Card.Body style={{ padding: '0px', backgroundColor: '#f0f0f1' }}>
```

Comment:
Let's avoid inline styling when possible. The Bootstrap class for zero padding is `p-0`.